### ISSUE TITLE
Wrong parentFile fileName for mzXML in Windows

### ISSUE THREAD

**Issue by JosuaCarl** [ProteoWizard/pwiz#3215]
The mzXML files are saved with wrong file delimiters for the parent file, when using Windows. The conversion to mzML worked as expected with the correct delimiters.

With Windows:
``` 
                                                        !
<parentFile fileName="file:///D:\folder\tests\test_files/minimal_file.mzXML"
                fileType="processedData"
                fileSha1="4d590e5e85e6ae2eb812f076aa2cef64e0612be5"/>
```

With Linux:
```
<parentFile fileName="file:////mnt/d/folder/tests/test_files/minimal_file.mzXML"
                fileType="processedData"
                fileSha1=""/>
```

I encountered the error when testing, whether the correct input path is saved in the file.

**Comment by chambm** [comment]
Which part is wrong? The final forward slash after \test_files? Because backslashes in Windows are correct (but forward slashes work too; mixed slashes can be a problem though).

**Comment by JosuaCarl** [comment]
The problem is that it's mixed, making it hard to parse automatically. It starts with / and ends with \. See the `!` where I marked the slash.

Sidenote: Generating a `.mzML` file always uses the OS's slashing flavor.
